Mixer.Base/Model/Interactive/InteractiveOnParticipantJoinModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mixer.Base.Model.Interactive
+{
+   public class InteractiveOnParticipantJoinModel
```

Comment:
There's no need to make multiple classes for the same data that lies at its core. You can remove OnLeave and OnUpdated models and just make a "InteractiveParticipantsChangedModel"